src/components/App.js

```diff
@@ -56,20 +114,31 @@ function App() {
     const displayProducts = getItemsForDisplay()
     setItemsForDisplay(displayProducts)
   }, [items]);
-  
-
 
   return (
     <div className="app">
       {!loading && 
         <>
-          <Landing />
-          <Products itemsForDisplay={itemsForDisplay} />
-          <About />
+          <NavLink to='/shopping-bag'>Cart</NavLink>
```

Comment:
Non blocker but could this Navlink be moved to a new component called Navbar?

Reply:
This is just a placeholder while waiting for the Navbar. It can be deleted. Sorry for not leaving a comment.
Jamie says he is planning to get to NavBar tomorrow. I will pick it up if he ends up being too swamped, since he will have a lot to test on the landing page.

Reply:
Perfect! Thanks for clarifying